packages/expo-cli/src/commands/utils/npm.ts

```diff
@@ -69,42 +73,37 @@ export async function npmViewAsync(...props: string[]): Promise<JSONValue> {
   }
 }
 
-/** Given a package name like `expo` or `expo@beta`, return the registry URL if it exists. */
-export async function getNpmUrlAsync(packageName: string): Promise<string> {
-  const results = await npmViewAsync(packageName, 'dist.tarball');
-
-  assert(results, `Could not get npm url for package "${packageName}"`);
-
-  // Fully qualified url returns a string.
-  // Example:
-  // 𝝠 npm view expo-template-bare-minimum@sdk-33 dist.tarball --json
-  if (typeof results === 'string') {
-    return results;
-  }
-
-  // When the tag is arbitrary, the tarball url is an array, return the last value as it's the most recent.
-  // Example:
-  // 𝝠 npm view expo-template-bare-minimum@33 dist.tarball --json
-  if (Array.isArray(results)) {
-    return results[results.length - 1] as string;
-  }
-
-  throw new CommandError(
-    'Expected results of `npm view ...` to be an array or string. Instead found: ' + results
-  );
-}
-
 // @ts-ignore
 const pipeline = promisify(Stream.pipeline);
 
 export async function downloadAndExtractNpmModuleAsync(
   npmName: string,
   props: ExtractProps
 ): Promise<void> {
-  const url = await getNpmUrlAsync(npmName);
-
-  Log.debug('Fetch from URL:', url);
-  await extractNpmTarballFromUrlAsync(url, props);
+  Log.debug(`Looking for tarball for ${npmName} in ${getCacheFilePath()}...`);
+  try {
+    const cachePath = getCacheFilePath();
```

Comment:
you need to ensure that `cachePath` exists, that is why those tests are failing

Reply:
Doh! Thanks!